deploy/injector-deployment.yaml

```diff
@@ -32,6 +32,8 @@ spec:
               value: ":8080"
             - name: AGENT_INJECT_LOG_LEVEL
               value: "info"
+            - name: AGENT_INJECT_LOG_FORMAT
+                - value: "json"
```

Comment:
Format is off here.

Reply:
I don't think he default should be `json`.  It's nice that there's a configurable for this now, however, not everyone is going to want JSON by default.

Reply:
I agree, default log settings should be similar to [hashicorp/vault](https://github.com/hashicorp/vault). Fixed.